caluma/extensions/events/form.py

```diff
@@ -0,0 +1,122 @@
+import csv
+import io
+import logging
+
+from django.db import transaction
+from django.db.models.signals import post_save
+
+from caluma.caluma_core.events import on
+from caluma.caluma_form import models as caluma_form_models
+
+from ..settings import settings
+
+logger = logging.getLogger(__name__)
+
+
+@on(post_save, sender=caluma_form_models.AnswerDocument, raise_exception=True)
+@transaction.atomic
+def update_table_summary(instance, *args, **kwargs):
+    """
+    Update table summary.
+
+    If a question within a table's row form has the `meta` attributes
+    `summary-question` and `summary-mode` set, then upon save, a summary
+    will be written, as follows:
```

Comment:
According to our call just before, this docstring fragment is incorrect and should be rewritten:

```suggestion
    If a table question has the `meta` attributes `summary-question` and
    `summary-mode` set, then upon save of a table row (or answer within),
    a summary of the table will be written, as follows:
```

Reply:
Done.